components/dashboard/src/components/start-workspace.tsx

```diff
@@ -354,6 +354,9 @@ export class StartWorkspace extends React.Component<StartWorkspaceProps, StartWo
         if (isError) {
             message = <div className="message action">
                 <Button className='button' variant='outlined' color='secondary' onClick={() => this.redirectToDashboard()}>Go to Workspaces</Button>
+                <Button className='button' variant='outlined' color='secondary' onClick={() => {
+                  this.startWorkspace(this.props.workspaceId, true, true);
+                }}>Open Default Workspace</Button>
```

Comment:
This button should only be shown when an image build failed or an image is not available.
The label could be "Start with Default Docker Image".